R/logRatioPlot.R

```diff
@@ -1,253 +1,565 @@
-#' @param barSize Set the bar size (thickness of each bar perimeter; Default = 0.1)
-#' @param barWidth Set the bar width (Default = 0.8)
-#' @param barAlpha Transparency for the bar layer (Default = 1)
-#' @param pointColor Color for the point layer (Default = "grey30")
-#' @param pointFill Fill color for the point layer (Default = "dodgerblue4")
-#' @param pointShape Shape for the point layer (Default = 21; fillable circle)
-#' @param pointAlpha Transparency for the box layer (Default = 1)
-#' @param pointSize Size of the points (Default = 4)
-#' @param lineLayer Add a fitted line layer (Default = FALSE)
-#' @param lineColor Color of the line fit (Default = "dodgerblue4")
-#' @param lineSize Size of the line fit (Default = 1)
-#' @param lineFit Type of fit to use.  One of c("auto", "lm", "glm", "gam",
-#'   "loess"). (Default = "loess")
-#' @param lineType One of c("solid", "dashed", "dotted", "dotdash", "longdash",
-#'   "twodash"). (Default = "solid")
-#' @param baseFontSize The smallest size font in the figure in points. (Default =
-#'   12)
-#' @param themeStyle "bw" or "grey" which correspond to theme_bw or theme_grey
-#'   respectively. (Default = "bw")
+#' @param yColname Define the column name for the output of the boxplots (default = "logFC")
+#' @param CI.R_colname Define name of the CI high value (default = "CI.R")
+#' @param CI.L_colname Define name of the CI low value (default =  "CI.L")
+#' @param plotCategory One of "bar" or "point" (default = "bar")
+#' @param refLine Adds a horizontal line at y = 0 (default = TRUE)
+#' @param refLineColor Color for the reference line (default = "red")
+#' @param xlab X axis label (defaults to xColname)
+#' @param ylab Y axis label (defaults to yColname)
+#' @param title Plot title, set to NULL to disable (Optional, default is NULL)
+#' @param barColor Color for the bar outline (default = "dodgerblue4")
+#' @param barTransparency Transparency for the bar layer (default = 1)
+#' @param pointColor Color for the point layer (default = "grey30")
+#' @param pointShape Shape for the point layer, for ggplot plotType (default = "circle")
+#' @param pointTransparency Transparency for the box layer (default = 1)
+#' @param pointSize Size of the points, for ggplot plotType (default = 4)
 #' @param facet Specifies whether to facet (TRUE) or print individual plots
-#'   (FALSE)  (Default = TRUE)
-#' @param facetCol Explicitly set the number of rows for the facet plot. Default
-#'   behavior will automatically set the columns. (Default = ceiling(sqrt(length(unique(contrastsDF[facetCol])))))
-#' @param xAngle Angle to set the sample labels on the X axis (Default =  45; Range = 0-90)
-#' @param scales Specify same scales or independent scales for each subplot (Default = "free_y";
-#'   Allowed values: "fixed", "free_x", "free_y", "free")
+#'   (FALSE)  (default = TRUE)
+#' @param facetCol Explicitly set the number of rows for the facet plot. default
```

Comment:
Connie had me remove this for the obsplot. You can use the default value.

Reply:
done